controllers/datamovement_controller.go

```diff
@@ -192,10 +217,25 @@ func (r *DataMovementReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, handleInvalidError(err)
 	}
 
+	// Pull configurable values from the ConfigMap
+	var dmCmd string
+	var progressCollectInterval time.Duration
+	var numProcesses int
+	configMap := &corev1.ConfigMap{}
+	if err := r.Get(ctx, types.NamespacedName{Name: DMConfigMapName, Namespace: DMConfigMapNamespace}, configMap); err != nil {
+		log.Info("Config map not found - requeueing")
+		return ctrl.Result{Requeue: true}, nil
```

Comment:
Careful about squashing the err here; consider the time the RBAC rules were wrong, you would constantly think the config map was not found but further investigation would reveal that error is a permissions type. Since "not found" is really an exceptional case, you could just return `err` and not get cheeky with Requeue. Might dirty the logs during the unit test, but that's OK. (Yes, I know to requeue was my suggestion)

Reply:
I'll remove the requeue and return the err